truss/patch/calc_patch.py

```diff
@@ -179,7 +179,7 @@ def _under_unsupported_patch_dir(path: str) -> bool:
 def _is_requirements_file_change(changed_path: str, new_config: TrussConfig):
     return (
         new_config.requirements_file
-        and Path(changed_path) == new_config.requirements_file
+        and Path(changed_path) == Path(new_config.requirements_file)
```

Comment:
the naming of of this method is confusing, and maybe merits a comment

Reply:
updated naming, added comments